libqtile/widget/generic_poll_text.py

```diff
@@ -92,3 +92,47 @@ def poll(self):
             text = "Can't parse"
 
         return text
+
+    
+class GenPollCommand(base.ThreadPoolText):
+    """A generic text widget to display output from scripts or shell commands"""
+
+    defaults = [
+        ("update_interval", 60,"update time in seconds"),
+        ("cmd", "date", "command line as a string or list of arguments to execute"),
```

Comment:
```suggestion
        ("cmd", None, "command line as a string or list of arguments to execute"),
```
I wouldn't have a default value here.